src/Symfony/Component/OptionsResolver/Tests/Debug/OptionsResolverIntrospectorTest.php

```diff
@@ -241,7 +275,7 @@ public function testGetDeprecationMessageThrowsOnNoConfiguredValue()
         $resolver->setDefined('foo');
 
         $debug = new OptionsResolverIntrospector($resolver);
-        $this->assertSame('bar', $debug->getDeprecationMessage('foo'));
+        $this->assertSame('bar', $debug->getDeprecation('foo'));
```

Comment:
Shouldn't the returned value be an array?

Reply:
That's right, nice catch

Reply:
I just reread the test. I think the real issue here is that we shouldn't use `assertSame()` at all. It will never be executed as the `getDeprecation()` call triggers an exception, right?

Reply:
Right, that's why it worked before changing it to an array